code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
         <H3 font="secondary">My profile</H3>
       </GridCell>
+      
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+      </GridCell>
     </Grid>
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
+        <img
+          src={`${APP_URL}//images/stock/men/4.jpg`}
+          style={{ 
+            marginBottom: '2em', 
+            height: '24em', 
+            boxShadow: level1, 
+            borderRadius: '10px'
+          }}
+        /> 
+        <H4>{props.user.details.name}</H4>
+      </GridCell>
+
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        <H4 font='secondary' style={{ marginBottom: '1.5em' }}>Account Details</H4>
 
+        <H6 style={{ marginBottom: '0.5em' }}>Account email:</H6>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
 
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
+        <H6 style={{ marginBottom: '0.5em' }}>Shipping Address:</H6>
+        <p style={{ color: grey2, marginBottom: '2em' }}>
+          123 Vanilla St.<br/>Chocolate, CO<br/>80911
+        </p>
 
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+        <H6 style={{ marginBottom: '0.5em' }}>Bio:</H6>
+        <p style={{ color: grey2, width: '60%', margin: '0 auto 2em' }}>
+          "Clothes are just accessories for my rollerblades, and I'm all about accessories."
+        </p>
+
+        <Button theme="primary">Edit Profile Info</Button>
+      </GridCell>
+
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
```

Comment:
Looks just like the wire frames, I am so impressed by all of us. Great work Team!